docs/concepts/privacy.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Privacy
+sidebarDepth: 0
+description: Learn about user privacy in IPFS and why it does not come with a built-in privacy layer or encryption.
+related:
+  'Article: IPFS Privacy (Textile)': https://medium.com/pinata/ipfs-privacy-711f4b72b2ea
+---
+
+# IPFS and privacy
+
+As you explore IPFS and learn about its core concepts, it's important to know that as a protocol for peer-to-peer data storage and delivery, IPFS in itself does not explicitly have a "privacy layer" built-in. This is in line with key principles of protocol design since different uses of IPFS over its life may call for different approaches to privacy. Explicitly implementing an approach to privacy within the IPFS core could "box in" future builders and implementers due to a lack of modularity, flexibility, and future-proofing.
```

Comment:
"with key principles of protocol design"

"of the protocol design" ? or do you mean protocol design generally? (seems a strong assertion)

Reply:
This paragraph leads with "not built in", but I'd prefer to flip that negative to a positive as you do later in the paragraph.

Reply:
Rearranged and clarified in https://github.com/ipfs/ipfs-docs/pull/554/commits/d387be8909eae4c3ce2166367505952c741f549d